servicenotifier/doc/adoc/servicenotifier.adoc

```diff
@@ -0,0 +1,43 @@
+= ServiceNotifiers Service
+
+Die ServiceNotifier-Extension bietet eine Möglichkeit, Dienste beim CAS zu registrieren.
+Sie bietet den Vorteil, dass auf Veränderungen bzw. Updates auf Tabellen gehorcht werden kann und dementsprechende Dienste benachrichtigt werden können.
+
+[plantuml, format="svg", title="title"]
+.Caption
+....
+@startuml
+start
+:Anfrage;
+:SQL-Procedure-Controller;
+if (Ist Erweiterung vorhanden?) then (ja)
+:Erweiterung ausführen.;
+else (nein)
+:SQL-Prozedur ausführen.;
+endif
+if (Ist ServiceNotifier vorhanden?) then (ja)
+:Betroffene Dienste informieren.;
+endif
+:Result auf Anfrage ausgeben.;
+stop
+@enduml
+....
+
+WARNING: Die ServiceNotifiers werden nach den Extensions ausgeführt, da Extensions ebenfalls Veränderungen auf Tabellen auslösen können.
+
+==== ServiceNotifier API
+
+Die zwei wichtigsten Objekte sind die beiden Maps, welche die Informationen zum Dienst und den Prozeduren enthalten:
```

Comment:
Da wir die ServiceNotifierService erstmals deprecated haben,
sollte man diese auch nicht in der Doku erwähnen.
Dass wird nur zu Verwirrungen sorgen.